Gemfile

```diff
@@ -8,18 +8,14 @@ gemspec
 gem 'json', :git => 'https://github.com/segiddins/json.git', :branch => 'seg-1.7.7-ruby-2.2'
 
 group :development do
-  gem 'activesupport', '>= 4.0.2', '!= 4.2.11.12', '< 5' # Pinned < 5 to ensure we're speccing 4.x.x
-  gem 'public_suffix', '>= 2.0.5', '< 3' # pinned since 3+ drops support for Ruby 2.0
-  gem 'minitest', '5.12.0' # Pinned since > 5.12.0+ drops support for Ruby 2.0
   gem 'bacon'
-  gem 'mocha'
+  gem 'mocha', '< 1.5'
```

Comment:
Was hitting https://github.com/rails/protected_attributes/issues/93, tried some of the workarounds mentioned there but didn't have luck